xradar/georeference/transforms.py

```diff
@@ -95,6 +98,10 @@ def get_x_y_z(ds, earth_radius=6371000, effective_radius_fraction=None):
     .. [1] Doviak and Zrnic, Doppler Radar and Weather Observations, Second
         Edition, 1993, p. 21.
     """
+    if earth_radius is None:
```

Comment:
@mgrover1 Actually it is written in any case, when georeferenced. So the user doesn't have to do anything, beside georeferencing.